spec/cassettes/statistics/statistics.yml

```diff
@@ -10,9 +10,9 @@ http_interactions:
       User-Agent:
       - Faraday v0.9.2
       X-Cassieq-Request-Time:
-      - '2016-04-12T02:03:11Z'
+      - '2016-04-18T19:22:53Z'
       Authorization:
-      - Signed smxpy9my1EIInq6nKc6Fcig5Ws2kjJ5-1jq0nMNVBBg
+      - Signed GdzpbmzPdzpXyb-Pdy45IPi7_dYB9rBiKf1jY50JWLw
```

Comment:
No sweat, its all about iteration. I'm just glad you have solid tests :) Thats more than most!  Thanks for the hard work tom!